scripts/scil_volume_pairwise_comparison.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of masks and atlas.
+All volumes must be in the same space.
+
+For the voxel representation, the computed similarity measures are:
```

Comment:
You copied the text from the other script. But here, you don't have any other representation, so it sounds a little weird to have "For the voxel representation"